README.md

```diff
@@ -30,8 +30,7 @@ cargo run -p with_winit
 
 ### SVG
 
-This repository also includes [`vello_svg`](./integrations/vello_svg/), which supports converting 
-a [`usvg`](https://crates.io/crates/usvg) `Tree` into a Vello scene.
+This repository also includes [`vello_svg`](./integrations/vello_svg/), which supports converting a [`usvg`](https://crates.io/crates/usvg) `Tree` into a Vello scene.
```

Comment:
Can someone help explain why GitHub thinks this is different? It's not clear on mobile (android)

Reply:
look at the line numbers, a newline was removed